llvm/test/ExecutionEngine/JITLink/x86-64/ELF_vtune.s

```diff
@@ -0,0 +1,52 @@
+# REQUIRES: native && x86_64-linux && intel-jitevents
+
+# RUN: rm -rf %t && mkdir -p %t
+# RUN: llvm-mc -triple=x86_64-unknown-linux -position-independent \
```

Comment:
remove `-position-independent`. Only mips/sparc use the option for some assembly design flaws.

Reply:
Fixed!